util/design/lib/OtpMemMap.py

```diff
@@ -355,3 +382,20 @@ def create_digests_table(self):
                         headers="firstrow",
                         tablefmt="pipe",
                         colalign=colalign)
+
+    def get_part_idx(self, part_name):
+        ''' Get partition index, return -1 if it does not exist'''
+        if part_name in self.part_index:
+            return self.part_index[part_name]['index']
+        else:
+            return -1
```

Comment:
Not that it really matters, but there's a slightly neater way to do this sort of thing:
```python
        part_index = self.part_index.get(part_name)
        return -1 if part_index is None else part_index['index']
```
It's a bit shorter and avoids the double lookup in the dict (which, admittedly, is unlikely to take long here!)

For `get_item_index` below, the "if expression" gets a bit longer and needs splitting, but you can still do something like this:
```python
        part_index = self.part_index.get(part_name)
        return (-1 if part_index is None
                else part_index['items'].get(item_name, -1))
```

Reply:
That's pretty neat, thanks for the suggestion!